addons/project_mail_plugin/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Project Mail Plugin',
+    'version': '1.0',
+    'category': 'Services/Project',
+    'sequence': 5,
+    'summary': 'Turn emails received in your mailbox into tasks and log their content as internal notes.',
```

Comment:
Note sure but I think summary is displayed in kanban cards. Isn't it a bit long ?

Reply:
Okay, changed to something much shorter!